File: migrate.py
import os
import postgres

db_url = "postgresql://postgres:JXaYfLedIWpwfXXOFuRkhSityLMAfole@crossover.proxy.rlwy.net:40549/railway"

try:
    sql = postgres.connect(db_url)
    print("Conectado ao banco de dados.")
    
    # Adicionar colunas manualmente
    sql.execute('ALTER TABLE users ADD COLUMN IF NOT EXISTS "lastName" text;')
    print("Coluna 'lastName' adicionada ou já existente.")
    
    sql.execute('ALTER TABLE users ADD COLUMN IF NOT EXISTS "phone" varchar(20);')
    print("Coluna 'phone' adicionada ou já existente.")
    
    sql.close()
    print("Migração concluída com sucesso.")
except Exception as e:
    print(f"Erro durante a migração: {e}")

File: migrate.js
import postgres from './server/node_modules/postgres/src/index.js';

const sql = postgres('postgresql://postgres:JXaYfLedIWpwfXXOFuRkhSityLMAfole@crossover.proxy.rlwy.net:40549/railway');

async function migrate() {
  try {
    console.log('Conectando ao banco de dados...');
    
    await sql`ALTER TABLE users ADD COLUMN IF NOT EXISTS "lastName" text;`;
    console.log("Coluna 'lastName' adicionada ou já existente.");
    
    await sql`ALTER TABLE users ADD COLUMN IF NOT EXISTS "phone" varchar(20);`;
    console.log("Coluna 'phone' adicionada ou já existente.");
    
    console.log('Migração concluída com sucesso.');
  } catch (error) {
    console.error('Erro durante a migração:', error);
  } finally {
    await sql.end();
  }
}

migrate();
